src/swig/python/CMakeLists.txt.in

```diff
@@ -0,0 +1,25 @@
+cmake_minimum_required(VERSION 3.18...3.22)
+
+project(_nlopt)
+
+find_package(PythonExtensions REQUIRED)
+find_package (NumPy REQUIRED)
+find_package (SWIG 3)
+include (UseSWIG)
+
+# Find NLopt C++ package
+set (NLopt_ROOT @CMAKE_CURRENT_BINARY_DIR@/../../)
+find_package (NLopt CONFIG REQUIRED)
+
+set (SWIG_MODULE__nlopt_EXTRA_DEPS nlopt-python.i numpy.i)
```

Comment:
Couldn't this depend on `../nlopt-python.i` etcetera if the paths are set correctly?

Reply:
I spent quite some time trying to set up paths correctly, but couldn't figure out how to do it.